package.json

```diff
@@ -57,6 +57,6 @@
     "webpack-cli": "^4.10.0"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "3.1.0"
+    "@eppo/js-client-sdk-common": "3.2.1"
```

Comment:
Build should pass once [this PR is merged and released.](https://github.com/Eppo-exp/js-client-sdk-common/pull/80)